imod/tests/test_mf6/test_mf6_regrid.py

```diff
@@ -115,9 +116,13 @@ def test_regrid_structured():
     """
     structured_grid_packages = create_package_instances(is_structured=True)
     new_grid = grid_data_structured(np.float64, 12, 2.5)
+    old_grid = grid_data_structured (np.float_, 1.0e-4, 5.0)
```

Comment:
More consistent with declaration ``new_grid``:

```suggestion
    old_grid = grid_data_structured(np.float64, 1.0e-4, 5.0)
```